Dockerfile

```diff
@@ -50,6 +51,8 @@ RUN curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/local
 	&& /usr/local/bin/composer global require hirak/prestissimo \
 	&& /usr/local/bin/composer create-project playbloom/satisfy:dev-master --stability=dev \
 	&& chmod -R 777 /satisfy \
+	&& rm -rf /satisfy/vendor/composer/satis \
+	&& cd /satisfy && composer update -o "composer/satis" \
```

Comment:
Why do you need those two lines ?

Reply:
Because the locked Version of satis in playbloom/satisfy is some times to old. And don't work with new Gitlab Version.